src/Uno.UI/UI/Xaml/Controls/Canvas/Canvas.cs

```diff
@@ -29,7 +29,7 @@ public static void SetLeft(DependencyObject obj, double value)
 			=> SetLeftValue(obj, value);
 
 
-		[GeneratedDependencyProperty(DefaultValue = 0.0d, AttachedBackingFieldOwner = typeof(UIElement), Attached = true)]
+		[GeneratedDependencyProperty(DefaultValue = 0.0d, AttachedBackingFieldOwner = typeof(UIElement), Attached = true, Options = FrameworkPropertyMetadataOptions.AutoConvert)]
```

Comment:
Will this fix #3622? That was reported for the syntax `SetValue(Windows.UI.Xaml.Controls.Canvas.LeftProperty, 42)`

Reply:
Yes, I tagged the wrong issue, my bad.